### PR TITLE
Displaying uco price and prompting user confirmation

### PR DESCRIPTION
**Pull request by manujuniris**
### **Description** 

This PR displays the transaction fee, rates (USD & EUR) and asks a user to confirm sending a transaction or not. 

The following output was captured - 

Screenshot from 2022-03-28 14-17-43

### CODE REVIEW

**Review by samuel-uniris — CHANGES_REQUESTED** [review]
I think you can display the fee and rate in the confirmation .

For example: 
"The transaction would cost X UCO ( $Y). Do you confirm? "

**Comment by manujuniris** [comment]
> I think you can display the fee and rate in the confirmation .
> 
> For example: "The transaction would cost X UCO ( $Y). Do you confirm? "

Done

File: commands/deploy_folder.js [hunk 1]
```diff
@@ -102,8 +103,12 @@ exports.handler = async function (argv) {
...
 
         try {
-            const { fee: fee } = await archethic.getTransactionFee(transaction, argv.endpoint)
-            console.log(chalk.yellow("Transaction fee : " +fee))
+            const { fee: fee, rates: rates } = await archethic.getTransactionFee(transaction, argv.endpoint)
+            
+
+            await yesno({
+                question:  chalk.yellow('The transaction would cost ' +fee+ ' UCO ($ ' +rates.usd+ ' € ' +rates.eur+ '). Do you want to confirm ?')
+            });
```
Comment (blackode) [inline commands/deploy_folder.js:111]: I don't see you are saving the user input here. I mean, If he/she type `yes` or `no`. 
I don't see the logic for yes and no sections here...

If he/she press no means It should not allow transaction right? If yes, then it is not happening.

I feel like we are just prompting the message but not considering the user input any how.

File: commands/deploy_website.js [hunk 2]
```diff
@@ -107,8 +108,11 @@ exports.handler = async function (argv) {
             .originSign(originPrivateKey)
 
         try {
-            const { fee: fee } = await archethic.getTransactionFee(transaction, argv.endpoint)
-            console.log(chalk.yellow("Transaction fee : " +fee))
+            const { fee: fee, rates: rates } = await archethic.getTransactionFee(transaction, argv.endpoint)
+            
+            await yesno({
+                question:  chalk.yellow('The transaction would cost ' +fee+ ' UCO ($ ' +rates.usd+ ' € ' +rates.eur+ '). Do you want to confirm ?')
+            });
```
Comment (blackode) [inline commands/deploy_website.js:115]: same here too...

File: commands/deploy_website.js [hunk 3]
```diff
@@ -190,8 +194,12 @@ exports.handler = async function (argv) {
...
 
             try {
-                const { fee: fee } = await archethic.getTransactionFee(transaction, argv.endpoint)
-                console.log(chalk.yellow("Transaction fee : " +fee))
+                const { fee: fee, rates: rates } = await archethic.getTransactionFee(transaction, argv.endpoint)
+               
+
+                await yesno({
+                    question:  chalk.yellow('The transaction would cost ' +fee+ ' UCO ($ ' +rates.usd+ ' € ' +rates.eur+ '). Do you want to confirm ?')
+                });
```
Comment (blackode) [inline commands/deploy_website.js:202]: here as well...

**Review by blackode — CHANGES_REQUESTED** [review]
- [x] Read PR description
- [x] Read AC
- [x] Pulled branch, manually tested, verified AC are met
- [x] Reviewed the code

I feel like evaluating and user input is not performed here...

Please check this.
image

**Review by blackode — COMMENTED** [review]
image
After the transaction is successful, It is not coming out of the process. Is that Normal ?
I mean It is not allowing me to enter one more command... 

I have to press Ctrl+c then have to submit again.    

Is this expected behavior?

**Review by blackode — APPROVED** [review]
- [x] Read PR description
- [x] Read AC
- [x] Pulled branch, manually tested, verified AC are met
- [x] Reviewed the code
- [x] Deploying File
image

- [x] Deploying Folder
image

- [x] Deploying website

**Comment by manujuniris** [comment]
> A comment regarding confirmation when we are deploying multiple files. You see the confirmation doesn't give much information about which file the transaction is about. Do you see a way to enhance it, to give more hint for fee and confirmation ?

Fixed by displaying "File deployed successfully", after sending transaction successfully.

**Review by samuel-uniris — APPROVED** [review]

**Pull request merged**